site/source/docs/tools_reference/emcc.rst

```diff
@@ -282,6 +282,9 @@ Options that are modified or new in *emcc* are listed below:
 ``--exclude-file <name>``
 	Files and directories to be excluded from :ref:`--embed-file <emcc-embed-file>` and :ref:`--preload-file <emcc-preload-file>`. Wildcards (*) are supported.
 
+``--fully-prepare-files``
+	Tells the file packager to not only load the preloaded files, but also prepare them, similarly to calling ``emscripten_async_prepare()``: This also does preparation tasks like decoding images using the browser codecs.
```

Comment:
"This also does preparation tasks like decoding images using the browser codecs.": what other preparation tasks are there than decoding assets? I thought that was the only thing?